pom.xml

```diff
@@ -128,22 +128,15 @@
         <scope>test</scope>
       </dependency>
       <dependency>
-        <!-- Needed as PlexusTestCase extends junit.framework.TestCase -->
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
-        <scope>test</scope>
-      </dependency>
-      <dependency>
-        <groupId>org.mockito</groupId>
-        <artifactId>mockito-core</artifactId>
-        <version>4.0.0</version>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Is it needed?

Reply:
Without `junit-jupiter-engine` the tests fail in both `enforcer-rules` and `maven-enforcer-plugin`.

I had run `dependency:tree`, which shows JUnit 4 is no longer on the classpath.

`dependency:analyze` shows `enforcer-rules` would only need `mockito-core` as opposed to `mockito-junit-jupiter`, but then you'd have to manage that dependency as well.